modules/core/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsConfigurationManager.java

```diff
@@ -165,11 +165,13 @@ public class IgniteStatisticsConfigurationManager {
             assert !F.isEmpty(cols);
 
             // Drop statistics after columns dropped.
-            dropStatistics(
-                Collections.singletonList(
-                    new StatisticsTarget(schemaName, typeDesc.tableName(), cols.toArray(EMPTY_STRINGS))
-                ),
-                false
+            mgmtBusyExecutor.execute(() ->
```

Comment:
Can a related statistic survive for long if the executor is busy for long by an active statistics collection tasks?